src/CairoLib.sol

```diff
@@ -3,155 +3,79 @@ pragma solidity >=0.8.0 <0.9.0;
 
```

Comment:
Most of the docstring of hyperloaded functions are gone, it there a reason for this ?

Reply:
yes, they're useless in my senses